pkg/kube/multicluster/secretcontroller.go

```diff
@@ -301,8 +301,7 @@ func (c *Controller) addSecret(name types.NamespacedName, s *corev1.Secret) erro
 		// We run cluster async so we do not block, as this requires actually connecting to the cluster and loading configuration.
 		c.cs.Store(secretKey, remoteCluster.ID, remoteCluster)
 		go func() {
-			remoteCluster.Run(c.meshWatcher, c.handlers, callback)
-			logger.Infof("finished callback for cluster")
```

Comment:
nit: can we add a log when the cluster is done syncing? It can be in `Run` if preferred

Reply:
make sense